rest-api/tools/deploy_app.sh

```diff
@@ -35,6 +35,12 @@ then
   usage
 fi
 
+if [[ `git status --porcelain` ]]; then
```

Comment:
Just to be extra safe, we may want to repeat this check below, at least before the deploy step (line 160).

That way if any files get modified while the DB or config is being updated, we will bail out before deploying.

Reply:
Er, sorry. This needs to run after line 107... up here the right version of the code hasn't been checked out yet.